### ISSUE TITLE
Error message "The parameter page is missing" shown if clicking on "Add new source" in the Repository Hierarchy list

### ISSUE THREAD

**Issue by Jefferson49** [Jefferson49/RepositoryHierarchy#22]
Observed with version 1.2.4 and webtrees 2.1.15

Screenshot.
![parameter page is missing](https://user-images.githubusercontent.com/81484983/210617515-0670354e-e693-4862-959c-33b0935c776c.JPG)

**Comment by Jefferson49** [comment]
See discussion in webtrees forum, thread [#37628](https://www.webtrees.net/index.php/en/forum/help-for-release-2-1-x/37628-error-message-the-parameter-page-is-missing-if-opening-a-modal)

**Comment by Jefferson49** [comment]
There seems to exist a dependency between the Repository Hierarchy module and the [Fancy Treeview](https://github.com/JustCarmen/webtrees-fancy-treeview) module (2.0.0-beta3 version). If both modules are installed in parallel, the issue occurs on some (but not on each) webtrees installations.

**Comment by Jefferson49** [comment]
The route of the Fancy Treeview module is defined as follows and seems to clash with the route of the Repository Hierarchy module:
/tree/{tree}/{module}/{xref}/{name}/{type}/{page}

If patching to the following route, the issue disappears:
/tree/{tree}/fancy_treeview/{module}/{xref}/{name}/{type}/{page}

**Comment by Jefferson49** [comment]
Fixed in the Fancy Treeview module

**Comment by LarsRabe** [comment]
@Jefferson49 
You might want to reopen this? see https://github.com/JustCarmen/webtrees-fancy-treeview/issues/115

**Comment by Jefferson49** [comment]
@LarsRabe, I think it is a misunderstanding. In the webtrees forum, two different topics were discussed in parallel threads nearly at the same time:
1. Robustness of custom modules against renaming of the module folder name
2. Error message "The parameter page is missing" 

I discussed 2.) (which is also issue #22, where we are currently typing in) with Carmen and it was resolved with her latest release. In my installation, the latest versions of **Fancy Treeview** and **Repository Hierarchy** run in parallel without any errors. Therefore, #22 does not need to be re-opened from my point of view.

For number 1.) I improved the robustness of Repository Hierarchy in the latest release. If renaming the folder of Fancy Treeview in my installation, the top level menu seems to disappear. However, no PHP errrors etc. show up.
